### ISSUE TITLE
Issues with RPC Full node Documentation

### ISSUE THREAD

**Issue by KryptomineCH** [Chia-Network/chia-docs#262]
referring to:
- https://docs.chia.net/full-node-rpc

there are multiple issues:
https://docs.chia.net/full-node-rpc#healthz
```
// Response
{
    "success": "true"
}
```
seems to be an incorrect response its not a proper boolean.

- https://docs.chia.net/full-node-rpc#get_recent_signage_point_or_eos
has no examples 
```
get_recent_signage_point_or_eos
Retrieves a recent signage point or end of slot.

Paramaters - one of:

sp_hash: The hash of the output for a signage point (if it's in the middle of a sub slot)
challenge_hash: The challenge_hash for the subslot (if it's an end of sub slot challenge)
```

- https://docs.chia.net/full-node-rpc#get_all_mempool_items
has an invalid json response example. The response is cut of early, the following needs to be appended:
```
"success": true
}
```

- https://docs.chia.net/full-node-rpc#get_routes
same issue as healthz, success boolean formatted inforrectly

in general the full node documentation seems to lack in comparison to the other endpoints which have much better examples like in:
https://docs.chia.net/full-node-rpc#get_fee_estimate

which is a newer function and seems to be of more recent formatting

**Comment by KryptomineCH** [comment]
suggested pull request for the json errors.
Update full-node.md #263

couldnt provide samples for [get_recent_signage_point_or_eos](https://docs.chia.net/full-node-rpc#get_recent_signage_point_or_eos) myself

**Comment by Rigidity** [comment]
Fixed in #322
